src/components/posts_list.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component, PropTypes} from 'react';
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {StyleSheet, Text, View} from 'react-native';
+
+import * as postActions from 'actions/posts';
+import ErrorText from 'components/error_text';
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        flexDirection: 'column',
+        alignItems: 'center',
+        paddingTop: 200,
+        backgroundColor: 'white'
+    }
+});
+
+const propTypes = {
+    posts: PropTypes.object.isRequired,
+    current_team_id: PropTypes.string.isRequired,
```

Comment:
I have a nuanced preference here: camel case for function/method names and snake case for other variables/properties. I've realized that's too nuanced for most people to buy into. As Mike said, I'm going against the grain. So I'm learning to love camel case everywhere on this project.
However 4-space indentation is another matter, but I'll keep quiet about that...